repose-aggregator/src/docs/asciidoc/architecture/performance.adoc

```diff
@@ -158,18 +156,29 @@ The Jenkins job will run the performance tests and publish the results automatic
 
 === Running a Test Manually
 
-[IMPORTANT]
-.TODO
-====
-* Add the link to the Puppet manifest.
-====
+If you have the appropriate permissions on the Repose Jenkins instance, you can run the https://jenkins.openrepose.org/job/performance-test/[performance-test] Jenkins job to manually start a single performance test.
+
+. In the https://jenkins.openrepose.org/job/performance-test/[performance-test] Jenkins job click on the https://jenkins.openrepose.org/job/performance-test/build?delay=0sec[Build with Parameters] link.
+. Populate the build parameters for the test you want to run.
+. Click Build.
+
+.performance-test build parameters
+[options="header,autowidth"]
+|===
+| Name          | Required | Description                                                                   | Example
+| perf_test     | Yes      | name of the performance test you wish to perform                              | uses-cases/simple
+| script_lang   | No       | name of the scripting language to use when running the filters/scripting test | python
```

Comment:
We had planned to to solve another issue, but Damien found a way to solve that problem in one line rather than lots of copy/pasta that would have separated the tests out.